examples/OpenSSL-asm/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2014, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+set(TESTING_CONFIG_OPT FILEPATH ${CMAKE_CURRENT_LIST_DIR}/config.cmake)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+# Need NASM on Windows to compile asm code
+if(WIN32)
+project(download-NASM)
+
+hunter_add_package(NASM)
```

Comment:
If this package needed for building OpenSSL then it should be part of OpenSSL build scheme.

Reply:
How do you want to integrate it in the OpenSSL build scheme ?

Reply:
Call `hunter_add_package(NASM)` inside build scheme when option is `ON`.

See Qt for example:
* https://github.com/ruslo/hunter/blob/v0.23.49/cmake/projects/Qt/schemes/url_sha1_qt.cmake.in#L9
* https://github.com/ruslo/hunter/blob/v0.23.49/cmake/projects/Qt/schemes/url_sha1_qt.cmake.in#L203-L209